nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -267,33 +265,33 @@ To list the history and status of migrations execute:
 
 `SELECT * FROM TimeoutsMigration_State`
 
-To list the status of timeouts for an a previous/in-progress run take the `MigrationRunId` from the query about and execute:
+To list the status of timeouts for a previous/in-progress run, take the `MigrationRunId` from the results of the previous query and execute:
 
 `SELECT * FROM TimeoutData_migration_{MigrationRunId}`
 
-This will show all the timeouts, to which batch they belong, and its status: `0=Pending`, `1=Staged` and `2=Completed`.
+The results include the batch number and its status: `0=Pending`, `1=Staged`, or `2=Completed`.
 
 ### NHibernate persistence
 
 The history and migrated data is always kept in the database.
 
-To list the history and status of migrations execute:
+To list the history and status of migrations:
 
 `SELECT * FROM MigrationsEntity`
 
-To list all the timeouts that were staged for migration, run the following query
+To list all the timeouts that were staged for migration:
 
 `SELECT * FROM StagedTimeoutEntity` 
 
-This will return all of the timeouts including batch number and status of that batch, `0=Pending`, `1=Staged` and `2=Completed`.
+The results include the batch number and its status: `0=Pending`, `1=Staged`, or `2=Completed`.
 
 ### Azure Storage persistence
 
-To list the history and status of migrations open up the `timeoutsmigrationtoolstate` Azure Table inside the Storage Explorer.
+To list the history and status of migrations open the `timeoutsmigrationtoolstate` Azure Table inside the Storage Explorer.
 
-The migrated timeouts as well as their status can be inspected in the `timeoutsmigration` Azure Table. This will show all the timeouts, to which batch they belong, and its status: `0=Pending`, `1=Staged` and `2=Completed`.
+The migrated timeouts and their status are contained in the `timeoutsmigration` Azure Table. This includes the batch number its status: `0=Pending`, `1=Staged`, or `2=Completed`.
 
-To list all timeouts that have been transferred from the endpoints timeout table into the `timeoutsmigration` table, execute the following query and adjust the partition key cut-off time accordingly. In order to avoid querying the additional timeout data inside the table, scope the partition key to cutoff time plus a hundred years.
+To list all timeouts that were transferred from the endpoints timeout table to the `timeoutsmigration` table, execute the following query and adjust the partition key cut-off time accordingly. To avoid returning additional timeout data, scope the partition key to the cutoff time plus one hundred years.
```

Comment:
> To avoid returning additional timeout data...

What is this "additional" timeout data? Can we be more descriptive? What are these rows that are more than 100 years out?

Reply:
@adamralph the problem here is not timeouts that are more than 100 years old the problem is how the persisted works and how querying azure table including its weird lexicographical ordering operates. We could be more specific but if we do we end up explaining the internal legacy design of the persisted and need to explain query semantics of azure table which I find not helpful for the purpose of this document